src/Rings/MPolyQuo.jl

```diff
@@ -1577,7 +1577,7 @@ julia> HC = gens(L[1]);
 
 julia> EMB = L[2]
 Map defined by a julia-function with inverse
-  from homogeneous component of graded multivariate polynomial ring in 5 variables over QQ of degree [2 1]
+  from homogeneous component of graded multivariate polynomial ring in 5 variables over QQ of degree Abelian group element [2, 1]
```

Comment:
this doesn't look nice

Reply:
I'd argue that this is the fault of the "decorated" MPoly code in Oscar.jl: it assumes that elements of the grading group print in "just the right way", which is fragile. IMHO this code should be using a special function (say `_print_grading(x)`) which prints the way it wants. Well, or else we must make a contract with the implementation of these GrpAb elements to always be printed in a certain way (but again I think that's too fragile, it is a hidden coupling between the two otherwise independent code bases)

Reply:
When we overhauled the printing of the abelian group elements, we forgot to add the compact printing. This is how it was *before* the PR:
```
julia> a = A([0, 0])
Element of A with components [0 0]

julia> [a a; a a]
2×2 Matrix{GrpAbFinGenElem}:
 [0 0]  [0 0]
 [0 0]  [0 0]
```
We will restore the old behaviour, which will fix the problem here.

Reply:
There is the `@show_special_elem` macro which would go in the direction of what @fingolfin is suggesting. But for now, we don't need it here.